galaxy_ng/app/api/ui/serializers/user.py

```diff
@@ -31,13 +32,20 @@ class Meta:
             'groups',
             'password',
             'date_joined',
-            'is_superuser'
+            'is_superuser',
+            'auth_provider',
         )
         extra_kwargs = {
             'date_joined': {'read_only': True},
             'password': {'write_only': True, 'allow_blank': True, 'required': False}
         }
 
+    def get_auth_provider(self, user):
+        if hasattr(user, 'social_auth'):
```

Comment:
the model has `auth_provider` field, should that field also be exposed here?

Reply:
Could you clarify what you mean here?